index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <title>Thump a Trump</title>
+    <link rel="stylesheet" href="normalize.css" type="text/css">
+    <link rel="stylesheet" href="menustyle.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css2?family=Rock+Salt&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header>
+        <h1>Thump a Trump</h1>
+    </header>
+    <div class="startBtn">
+        <button><img src="play-solid.svg" class="startImg">START</button>
```

Comment:
I would question why this needs to be a <button>. Seems like you are not using any of the button features of the button! If you had a <a> tag here instead, you could have fewer CSS styles to override and it would make more semantic sense what this image is meant for -> a link to the game screen!